### ISSUE TITLE
Problemas en la versión MAC

### ISSUE THREAD

**Issue by s-garciap** [sigdeletras/Spanish_Inspire_Catastral_Downloader#28]
Buenos días, he detectado sobre dos ordenadores MAC diferentes que el plugin se abre, permite la selección de municipios y elementos a descargar, pero al iniciar la descarga se queda bloqueado siempre en el 0%. Utilizamos la versión LTR de QGis ¿Alguna idea? ¿A alguien más le sucede?

**Comment by jesbrz** [comment]
Hola. Como puedes ver en [https://github.com/sigdeletras/Spanish_Inspire_Catastral_Downloader/issues/26](url) a mi me pasa lo mismo. No he encontrado ninguna solución y el desarrollador tampoco. Si encontráis algo, por favor, ponedlo por aquí. Ahora la versión anterior tampoco funciona.

**Comment by JuanBreva** [comment]
A día de hoy, en la versión 3.40 Bratislava, y bajo Mac OS Monterey, este problema persiste.

**Comment by josereal** [comment]
misco problema por aquí, parece que no va a tener solución. En windows funciona

**Comment by s-garciap** [comment]
Creo que el problema también se ha extendido a Windows. ¿Quizá ahora es además un problema de Catastro, y ATOM se ha desconfigurado?

**Comment by s-garciap** [comment]
Me corrijo a mi mismo. Al actualizar a la última versión del complemento en Windows hace que no haya problema en la descarga. Para usuarios de MAC, pueden utilizar la descarga manual, o automatizarla vía Python. Las url del servicio atom tienen esta estructura:
http://www.catastro.hacienda.gob.es/INSPIRE/{subcarpeta}/{provincia}/{municipio}/A.ES.SDGC.{tipo}.{municipio_codigo}.zip
donde:
- tipo y subcarpeta es: "CP": "CadastralParcels" o  "AD": "Addresses" o  "BU": "buildings"
- provincia, municipio y municipio codigo se obtienen de aquí: https://www.catastro.hacienda.gob.es/INSPIRE/CadastralParcels/ES.SDGC.CP.atom.xml (ej: 50 es la provincia de Zaragoza, 50900 el municipio_codigo y 50900-ZARAGOZA el municipio

Ejemplo para las descargas de las parcelas de Zaragoza:
http://www.catastro.hacienda.gob.es/INSPIRE/CadastralParcels/50/50900-ZARAGOZA/A.ES.SDGC.CP.50900.zip

**Comment by jesbrz** [comment]
Hola @s-garciap, gracias por el truco. He probado lo que dices en Mac, pero esa estructura solo funciona curiosamente con las parcelas catastrales (CP) pero no con los edificios (BU) ni con las direcciones (AD). He seguido tu ejemplo, pero si en el url sustituyo CP por BU me lleva a una página con este error `Se ha producido un fallo al intentar mostrar una página o archivo.
Rogamos disculpen las molestias.` Igual con AD.

**Comment by jesbrz** [comment]
@s-garciap Efectivamente, ese era el problema, no me había dado cuenta. Gracias de nuevo.

**Comment by josereal** [comment]
sigue sin funcionar, ¿como lo automatizais en Qgis?
